test/intern/functional/Keyboard.js

```diff
@@ -86,20 +90,20 @@ define([
 					.then(function (classNames) {
 						var arr = classNames.split(' '),
 							containsClass = (arr.indexOf('dgrid-focus') !== -1);
-						assert.ok(containsClass, 'the home key should move focus to the first element in the list');
+						assert(containsClass, 'the home key should move focus to the first element in the list');
 					})
 					.end();
 		};
 	}
 
 	test.suite('Keyboard functional tests', function () {
-		test.before(function () {
+		test.before(function (suite) {
 			// Get our html page. This page should load all necessary scripts
 			// since this functional test module runs on the server and can't load
 			// such scripts. Further, in the html page, set a global "ready" var
 			// to true to tell the runner to continue.
-			return this.remote
-				.get(require.toUrl('./Keyboard.html'))
+			return suite.remote
+				.get('dgrid/test/intern/functional/Keyboard.html')
```

Comment:
@jason0x43 this used to use `require.toUrl` - it seems `require` now runs in Node.js and returns a fully-qualified path from the file system, which of course fails in the browser. Is there a good way to be able to keep using a relative path (`./Keyboard.html`)?

Reply:
All relative `remote.get` calls are relative to the base path, which is the project root by default, so you should just be able to do `suite.remote.get('test/intern/functional/Keyboard.html')`.

Reply:
So relative paths are right out, then?

I had to set `basePath` to `'../'` in order for Intern's HTTP server to serve the sibling packages (`dojo, dijit, dstore`). Or is there another way to configure the root path of Intern's HTTP server?

Reply:
Relative paths are fine, it's just that they're relative to the basePath rather than to the importing module. If basePath is `../`, then `'dgrid/test/intern/functional/Keyboard.html'` sounds right.